docs/source/guide/minibatch-custom-sampler.rst

```diff
@@ -3,143 +3,133 @@
 6.4 Implementing Custom Graph Samplers
 ----------------------------------------------
 
-Implementing custom samplers involves subclassing the :class:`dgl.dataloading.Sampler`
-base class and implementing its abstract :attr:`sample` method.  The :attr:`sample`
-method should take in two arguments:
+Implementing custom samplers involves subclassing the
+:class:`dgl.graphbolt.SubgraphSampler` base class and implementing its abstract
+:attr:`_sample_subgraphs` method. The :attr:`_sample_subgraphs` method should
+take in seed nodes which are the nodes to sample neighbors from:
 
 .. code:: python
 
-   def sample(self, g, indices):
-       pass
+    def _sample_subgraphs(self, seed_nodes):
+        return input_nodes, sampled_subgraphs
```

Comment:
not related to this PR: we should add python doc here: https://github.com/dmlc/dgl/blob/aa9c11017e46c67c7964599bcbf6ba97f6f06f87/python/dgl/graphbolt/subgraph_sampler.py#L137C9-L137C26

Reply:
yes, it's missing